src/main/webapp/app/entities/feedback.model.ts

```diff
@@ -88,21 +96,46 @@ export class Feedback implements BaseEntity {
         if (!feedback.text) {
             return true;
         }
-        return !feedback.text.includes(STATIC_CODE_ANALYSIS_FEEDBACK_IDENTIFIER, 0) && !feedback.text.includes(SUBMISSION_POLICY_FEEDBACK_IDENTIFIER, 0);
+        return !feedback.text.startsWith(STATIC_CODE_ANALYSIS_FEEDBACK_IDENTIFIER) && !feedback.text.startsWith(SUBMISSION_POLICY_FEEDBACK_IDENTIFIER);
     }
 
     public static isStaticCodeAnalysisFeedback(that: Feedback): boolean {
         if (!that.text) {
             return false;
         }
-        return that.type === FeedbackType.AUTOMATIC && that.text.includes(STATIC_CODE_ANALYSIS_FEEDBACK_IDENTIFIER, 0);
+        return that.type === FeedbackType.AUTOMATIC && that.text.startsWith(STATIC_CODE_ANALYSIS_FEEDBACK_IDENTIFIER);
     }
 
     public static isSubmissionPolicyFeedback(that: Feedback): boolean {
         if (!that.text) {
             return false;
         }
-        return that.type === FeedbackType.AUTOMATIC && that.text.includes(SUBMISSION_POLICY_FEEDBACK_IDENTIFIER, 0);
+        return that.type === FeedbackType.AUTOMATIC && that.text.startsWith(SUBMISSION_POLICY_FEEDBACK_IDENTIFIER);
+    }
+
+    public static isFeedbackSuggestion(that: Feedback): boolean {
+        if (!that.text) {
+            return false;
+        }
+        return that.text.startsWith(FEEDBACK_SUGGESTION_IDENTIFIER);
```

Comment:
I'm changing them to always be `FeedbackType.MANUAL` because a lot of the code assumes that all `AUTOMATIC` / `AUTOMATIC_ADAPTED` feedback are automatically generated test cases, which does not make sense. We can detect feedback suggestions from the `FeedbackSuggestion:` prefix on the `text` anyway, so I think it makes sense to use `type = `FeedbackType.MANUAL`. I think using `AUTOMATIC_ADAPTED` would theoretically work, but would probably also introduce more confusion because normal suggestions would not be `AUTOMATIC`.

Reply:
Hmm, I think we have to think about overall feedback suggestions. I feel this is inconsistent and confusing compared to text exercises and potentially future supported exercise types. In text exercises suggestions from Athena are `AUTOMATIC` and when changed `AUTOMATIC_ADAPTED`. I think this generally makes sense. 

Are you suggesting that generally we should use `MANUAL` and `MANUAL_UNREFERENCED` for all exercise types where Athena is suggesting feedback and use a prefix for the `text`? Alternatively, we can also think about renaming `AUTOMATIC_ADAPTED` to `SUGGESTION_ADAPTED` and add a new feedback type called `SUGGESTION`.

But idk, I'm also fine with the idea where we are generally using the `text` field for all Athena feedback and its state 🤔

Reply:
Yes, I am suggesting to use `MANUAL`/`MANUAL_UNREFERENCED` + the `text` prefix 👍 

Otherwise, we either cannot distinguish suggestions from non-suggestions for programming exercises or we have to be inconsistent.

I am against renaming the feedback types because we should have to add a DB migration for millions of feedback that changes the enum value for the existing feedbacks.